yarn.lock

```diff
@@ -2307,12 +2307,25 @@
     call-me-maybe "^1.0.1"
     glob-to-regexp "^0.3.0"
 
+"@mui/core@^5.0.0-alpha.51":
```

Comment:
Wrong version linked probably, please check, we should not have this in the `yarn.lock`.

Reply:
seems like this is a mistake from data-grid. I have open an [issue](https://github.com/mui-org/material-ui-x/issues/3157) that will be fixed separately. For this PR, I will just trust what I got from `yarn`.